scorer/runner.py

```diff
@@ -42,25 +54,43 @@ def main():
         fname = os.path.join(config.SNAPSHOTS_PATH, snapshots[0])
         print(
             '{} - processing {} started ...'.format(str(datetime.now()).split('.')[0], fname))
+        restore_snapshot(fname)
         process(fname)
-        update_apps_verification()
+        update_apps_verification(fname)
```

Comment:
`fname` is not required to be passed to this function

Reply:
It's done.